src/flashbar/__integ__/pages/base.ts

```diff
@@ -22,9 +22,26 @@ export class FlashbarBasePage extends BasePageObject {
     return createWrapper().findFlashbar().findByClassName(selectors['notification-bar']).toSelector();
   }
 
+  getExpandButton() {
+    return createWrapper().findFlashbar().findByClassName(selectors.button).toSelector();
+  }
+
+  getDismissButton() {
+    return createWrapper().findFlashbar().findByClassName(selectors['dismiss-button']).toSelector();
+  }
+
   isFlashFocused(index: number) {
     return this.isFocused(
       flashbar.findItems().get(index).findByClassName(selectors['flash-focus-container']).toSelector()
     );
   }
+
+  getComputedStyles(selector: string, property: string, pseudoElement: string | null = null) {
```

Comment:
1. Did you try using the built-in method: https://webdriver.io/docs/api/element/getCSSProperty
1. If you tried and you are sure we need a custom one, let's move it to https://github.com/cloudscape-design/browser-test-tools/blob/main/src/page-objects/base.ts

Reply:
Native works!